apple-pie/index.html

```diff
@@ -5,17 +5,64 @@
-    all-purpose flour 1/4 cup water 3 tablespoons all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
-    brown sugar 8 Granny Smith apples - peeled, cored and sliced Directions Preheat oven to 425 degrees F (220 degrees
-    C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
-    to a boil. Reduce temperature and let simmer. Place the bottom crust in your pan. Fill with apples, mounded
-    slightly. Cover with a lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-    that it does not run off. Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-    degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
+    <header>
+      <h1>Apple Pie</h1>
+    </header>
+
+    <main>
+      <section id="intro">
+        <p>
+          This was my grandmother's apple pie recipe. I have never seen another
+          one quite like it. It will always be my favorite and has won me
+          several first place prizes in local competitions. I hope it becomes
+          one of your favorites as well!
+        </p>
+        <img src="./images/recipe-info.png" alt="recipe-info" class="image" />
+      </section>
+      <hr />
+
+      <section id="ingredients">
+        <h2 class="section-title">Ingredients</h2>
+        <ul class="text">
+          <li>1 recipe pastry for a 9 inch double crust pie</li>
+          <li>1/2 cup unsalted butter</li>
+          <li>3 tablespoons all-purpose flour</li>
+          <li>1/4 cup water</li>
+          <li>3 tablespoons all-purpose flour</li>
+          <li>1/4 cup water</li>
+          <li>1/2 cup white sugar</li>
+          <li>1/2 cup packed brown sugar</li>
+          <li>8 Granny Smith apples - peeled, cored and sliced</li>
+        </ul>
+        <img src="./images/cooking-info.png" alt="cooking-info" class="image" />
+      </section>
+      <hr />
+
+      <section id="directions">
+        <h2 class="section-title">Directions</h2>
+        <ol class="text">
+          <li>
+            1) Preheat oven to 425 degrees F (220 degrees C). Melt the butter in
+            a saucepan. Stir in flour to form a paste. Add water, white sugar
+            and brown sugar, and bring to a boil. Reduce temperature and let
+            simmer.
+          </li>
+          <li>
+            2) Place the bottom crust in your pan. Fill with apples, mounded
+            slightly. Cover with a lattice work crust. Gently pour the sugar and
+            butter liquid over the crust. Pour slowly so that it does not run
+            off.
+          </li>
+          <li>
+            3) Bake 15 minutes in the preheated oven. Reduce the temperature to
+            350 degrees F (175 degrees C). Continue baking for 35 to 45 minutes,
+            until apples are soft.
+          </li>
```

Comment:
Works ok like this, but there is also a way to create the list numbers like this "1 )" with CSS:
```
ol {
  counter-reset: list;
}

ol > li:before {
  content: counter(list) ") ";
  counter-increment: list;
}
```